### PR TITLE
1.Added in age field and hashedpassword for insert/addbean function. …

### PR DESCRIPTION
**Pull request by JKBrown11**
…2. Updating SQL because INSERT command is blocked on our uname group1_dev. [[[[PLEASE RUN NEW SQL FILE MoffatBayDB_NV3]]]] to update our priveleges to all tables. 
3. Current Status- DB receives new customer. The hashed pass is null and redirect fails.

### CODE REVIEW

File: Moffat_Bay_EW/src/main/java/mbTools/CustomerBean.java
Comment (JKBrown11) [inline Moffat_Bay_EW/src/main/java/mbTools/CustomerBean.java:1]: The get parameter wouldn't return a number for age; it returned only as a string, so I corrected the bean method.

File: Moffat_Bay_EW/src/main/java/mbTools/DataAccess.java
Comment (JKBrown11) [inline Moffat_Bay_EW/src/main/java/mbTools/DataAccess.java:1]: Storing only hashed password for security.

**Comment by JKBrown11** [comment]
Updated regEx for password. Updated a few other things here an there, be
en an afternoon of debugging. Successfully registered customer with full pw.

**Review by aumoore — APPROVED** [review]
reviewed changes and approving.

**Pull request merged**